blocks/market-commentary/market-commentary.js

```diff
@@ -0,0 +1,125 @@
+import { observe, Viewport } from '../../scripts/blocks-utils.js';
+import { fetchPlaceholders } from '../../scripts/aem.js';
+import { callMockCommenrtaryAPI } from '../../scripts/mockapi.js';
+import {
+  div, a, h4, p, span,
+} from '../../scripts/dom-builder.js';
+
+const placeholders = await fetchPlaceholders();
```

Comment:
I think we should fetch placeholders from `decorate` when they will be needed. Placing it here can potentially slow increase the script load time.